src/algorithm/distance.rs

```diff
@@ -10,10 +10,10 @@ use num_traits::pow::pow;
 pub trait Distance<T, Rhs = Self> {
     /// Returns the distance between two geometries
     ///
-    /// If a `Point` is contained by a `Polygon`, the distance is `0.0`  
```

Comment:
I think these spaces might have been intentional because markdown treats them as newlines:

http://spec.commonmark.org/0.26/#hard-line-breaks

I think with your changes, these sentences get put on the same line